CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Header table gets broken. [#409](https://github.com/scanapi/scanapi/pull/409)
```

Comment:
@bartvanherck the changelog generally has Pull Request ID instead of Issue ID. 

Feel free to amend the commit and update it. So you don't do two commits just for updating this. 

While amending the commit can you please also update the title of the commit to not have issue ID in it. We can have that in the bottom of the commit. 

Something like 👇🏾 

```
Fix: Break lines in column to prevent overflow

- Use break-all css in the right columns of the table to prevent overflow,
- Update the Changelog

Closes #409
```